src/System.Collections/tests/Performance/Perf.Dictionary.netcoreapp.cs

```diff
@@ -0,0 +1,473 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
I just haven't had a chance to update those yet. They are still on my list and should be removed in the next PR (just had to go grab some lunch).